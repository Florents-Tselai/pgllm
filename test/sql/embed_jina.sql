select array_length(llm_embed('hello world', 'jina-embeddings-v2-small-en'), 1);
select array_length(llm_embed('hello world', 'jina-embeddings-v2-base-en'), 1);
