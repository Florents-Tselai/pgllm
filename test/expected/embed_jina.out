select array_length(llm_embed('hello world', 'jina-embeddings-v2-small-en'), 1);
 array_length 
--------------
          512
(1 row)

select array_length(llm_embed('hello world', 'jina-embeddings-v2-base-en'), 1);
 array_length 
--------------
          768
(1 row)

